src/scripts/main.js

```diff
@@ -2,7 +2,21 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  for (let i = 0; i < 6; i++) {
+    row.append(document.createElement('td'));
+  }
+
+  row.children[0].innerText = person.name;
+  row.children[1].innerText = person.sex === 'm' ? 'Male' : 'Female';
+  row.children[2].innerText = person.born;
+  row.children[3].innerText = person.died;
+  row.children[4].innerText = person.died - person.born;
+  row.children[5].innerText = Math.ceil(person.died / 100);
```

Comment:
```suggestion
  row.cells[0].innerText = person.name;
  row.cells[1].innerText = person.sex === 'm' ? 'Male' : 'Female';
  row.cells[2].innerText = person.born;
  row.cells[3].innerText = person.died;
  row.cells[4].innerText = person.died - person.born;
  row.cells[5].innerText = Math.ceil(person.died / 100);
```
it's also possible to use 'cells' with tables, but it's up to you